airflow/jobs/scheduler_job_runner.py

```diff
@@ -264,27 +264,6 @@ def _debug_dump(self, signum: int, frame: FrameType | None) -> None:
         self.job.executor.debug_dump()
         self.log.info("-" * 80)
 
-    def is_alive(self, grace_multiplier: float | None = None) -> bool:
```

Comment:
This is not used any more - I missed during the refactor that this "special" case for SchedulerJob is not called any more.

Reply:
What do the healthchecks in the chart use? They used to call `SchedulerJob.get_latest_job().is_alive()`

Reply:
I worry that by removing this we'll break various installs.

What is the right thing to put in healthchecks/liveness probes now?

Probably needs a news fragment with updating instructions please